templates/cluster/eks/templates/awsmanagedcontrolplane.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: controlplane.cluster.x-k8s.io/v1beta2
+kind: AWSManagedControlPlane
+metadata:
+  name: {{ include "awsmanagedcontrolplane.name" . }}
+  namespace: hmc-system
+spec:
+  region: {{ .Values.region }}
+  {{- if not (quote .Values.sshKeyName | empty) }}
+  sshKeyName: {{ .Values.sshKeyName | quote }}
+  {{- end }}
+  version: {{ .Values.kubernetes.version }}
+  identityRef:
+    kind: {{ .Values.clusterIdentity.kind }}
+    name: {{ .Values.clusterIdentity.name }}
+  controlPlaneEndpoint:
+    host: {{ .Values.controlPlaneEndpointIP }}
+    port: 6443
```

Comment:
Why this is here? How user can fill it up if infrastructure is not created? 
Also it's not in the values, so the host will be blank.
I suggest removing this section completely.

Reply:
It fixes this error:
`E1003 16:23:13.909445       1 controller.go:329] "Reconciler error" err="failed to retrieve Spec.ControlPlaneEndpoint from infrastructure provider for Cluster \"eks-dev-slava-1\" in namespace \"hmc-system\": field not found" controller="cluster" controllerGroup="[cluster.x-k8s.io](http://cluster.x-k8s.io/)" controllerKind="Cluster" Cluster="hmc-system/eks-dev-slava-1" namespace="hmc-system" name="eks-dev-slava-1" reconcileID="0ac4ff1a-984b-47d4-8dd9-c273b8357998"`